tap_helpshift/streams.py

```diff
@@ -31,9 +31,12 @@ class Stream():
     def __init__(self, client=None, start_date=None):
         self.client = client
         if start_date:
-            self.start_date = start_date
+            self.start_date_int = start_date
```

Comment:
Renamed what was `self.start_date` to `self.start_date_int` and created a new `self.start_date` which is a datetime instance.